russian/fly/fly-where.md

```diff
@@ -0,0 +1,15 @@
+Как я мог об этом забыть? Я же птица!
+
+Вы с гордостью расправляете крылья и за несколько метров от земли падение останвливается
+и Вы всзмываете вверх.
+
+Вы дышите полной грудью! Свобода! Как это прекрасно! 
+
+Вы осматриваетесь по сторонам и решаете ...
+
+
+[ отправиться в пустыню](../desert/desert.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.